chia-bls/src/cached_bls.rs

```diff
@@ -0,0 +1,275 @@
+extern crate lru;
```

Comment:
I don't think this line is necessary. I don't recall ever having to use `extern crate`. ChatGPT suggests that since the rust 2018 edition you can import with `use` instead.